src/us/kbase/narrativejobservice/sdkjobs/DockerRunner.java

```diff
@@ -233,6 +234,29 @@ public static synchronized long getPidOfProcess(Process p) {
         return pid;
     }
 
+    public void runAlpineCleaner(File jobdir) throws Exception {
+        cl = createDockerClient();
+
+        String workdir = "/kb/module/work/*";
+        String alpineImageName = "alpine";
```

Comment:
Should we make a copy of the alpine container and put it into our kbase dockerhub repo?